locales/english.json

```diff
@@ -1,61 +1,41 @@
 {
   "common.cancel": "Cancel",
   "common.continue": "Continue",
-  "common.confirm": "Confirm",
   "common.documentTitle": "Data Marketplace",
   "auth.loading": "Signing you in\u2026",
   "auth.prompt": "Please sign in",
   "frontpage.hero.title": "Find and get access to the data you need, when you need it",
   "frontpage.hero.ingress": "Data Marketplace makes it easy to search for and request access to the data you need to do your job. Venenatis ipsum aliquet nibh at. Risus sapien erat vitae amet leo lobortis maecenas.",
   "frontpage.popularProductsHeader": "Popular",
   "frontpage.numberOfViews": "{numberOfViews} views",
-  "frontpage.seeMore": "See more",
-  "frontpage.placeholderSearch": "Search\u2026",
-  "frontpage.ariaSeeMore": "See more popular data products",
   "search.filterHeader": "Filter",
   "search.communitiesHeader": "Communitites",
   "search.statistic": "{count, plural, =0 {We were not able to find anything related to \u201C{searchTerm}\u201D} one {# result} other {# results}}",
   "search.view": "View",
   "search.lastUpdated": "Last updated on",
-  "page.authenticatedLoading": "Signing you in",
-  "page.unauthenticated": "Please sign in",
   "navbar.logo": "Equinor Data Marketplace",
-  "navbar.browse": "Browse",
-  "navbar.community": "Community",
   "navbar.placeholderSearch": "Search",
   "navbar.ariaUserMenu": "User menu",
-  "navbar.titleShoppingCart": "Shopping cart",
-  "navbar.taskStatistic": "Tasks: {remaining}",
-  "navbar.titleRefresh": "Refresh",
   "navbar.titleAccount": "Account",
-  "spinner.loading": "Please hold\u2026",
-  "asset.addToCart": "Add to cart",
   "asset.getAccess": "Get access",
   "asset.description": "Description",
   "asset.purpose": "Purpose",
   "asset.timeliness": "Timeliness",
   "asset.overview": "Overview",
   "asset.responsibilites": "Responsibilites",
-  "terms.ingress": "<p>These terms and conditions create a contract between you as a data consumer and the data provider (the “Agreement”).</p><p>When you are provided access to this Data Product, you will have to ensure that the following criteria are met:</p>",
+  "terms.ingress": "<p>The agreement between you and the data provider is created by these terms and use.</p><p>Access to this Data Product requires you to meet the following criteria.</p>",
```

Comment:
it's either "terms **of use**" or "terms **and conditions**". collibra calls it "terms and conditions", so i think it might be best if we use that too.

Reply:
Discussed this back and forth with Trond yesterday, what we wanted was to use the same wording in the label for the checkbox and the ingress. Since the label used "Terms of use" we changed the ingress. Doesn't matter for me which term to use, but I would argue that it should be consistent.

Reply:
Decided to use terms and conditions